components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/connector/RecoveryConfigImpl.java

```diff
@@ -338,4 +346,81 @@ public Properties getDefaultPropertyValues(String tenantDomain) throws IdentityG
         return null;
     }
 
+    private Property getPropertyObject(String type, String regex) {
```

Comment:
This is duplicated. Shall we move this to a Util?

Reply:
Fixed in commit 85062b1cef3140d703ce55348fce323acdfa482d